client/my-sites/customer-home/cards/tasks/reactivate-expired-plan/get-task.tsx

```diff
@@ -0,0 +1,63 @@
+import { translate } from 'i18n-calypso';
+import expiredIllustration from 'calypso/assets/images/customer-home/disconnected-dark.svg';
+
+export const getTask = ( context: any = {} ) => {
+	// TODO somehow we will need to decide which task details to return based on some context.
```

Comment:
The API doesn't have a way of passing context down with the view. I think should actually be two separate cards (from the API's point of view at least, you can implement the React components however you want). The same way the server returns different card types for `TASK_RENEW_**EXPIRED**_PLAN` and `TASK_RENEW_**EXPIRING**_PLAN`.

Reply:
Same goes for `revivalType === 3`; it should be another card. It is a bit repetitive, but the My Home API was intentionally designed that way to give the server the most control over the layout.

Reply:
Ok, I will set up three separate cards then on both the client and server.